ansys/dpf/core/meshed_region.py

```diff
@@ -207,6 +214,7 @@ def _get_unit(self):
         """
         return self._api.meshed_region_get_unit(self)
 
+    @clear_grid
```

Comment:
@cbellot000 Now that I think about it, do we actually want to reset the grid when changing the unit? Unless we add a mechanism to scale the mesh when the unit is changed, there is actually no benefit to this as I have not found any notion of unit in PyVista's UnstructuredGrid.
```suggestion
```